CMakeLists.txt

```diff
@@ -193,6 +193,8 @@ find_package(Boost 1.65 MODULE
     thread
   REQUIRED)
 
+include(ext/libpg_query_Ext.cmake)
```

Comment:
To be naming consistent - could we rename this file as `PGQuery_Ext` or `PG_Ext`?

Reply:
fixed.